tests/k6/common/dialog.js

```
@@ -155,6 +152,25 @@ export function setSystemLabel(dialog, systemLabel) {
     }
     dialog.systemLabel = systemLabel;
 }
+
+export function setServiceOwnerLabels(dialog, serviceOwnerLabels) {
+    if (!Array.isArray(serviceOwnerLabels) || serviceOwnerLabels.some(label => typeof label !== "string")) {
+        throw new Error("Invalid service owner labels provided");
+        console.log(serviceOwnerLabels);
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will throw an exception beforehand


(lint/correctness/noUnreachable)